.pylintrc

```diff
@@ -0,0 +1,577 @@
+[THINGS WE CARE ABOUT]
+
+[FORMAT]
+
+# Expected format of line ending, e.g. empty (any line ending), LF or CRLF.
+expected-line-ending-format=
+# Naming style matching correct class names.
+class-naming-style=PascalCase
+
+# Maximum number of nested blocks for function / method body
+max-nested-blocks=5
+
+# Maximum number of characters on a single line.
+max-line-length=90
+
+# String used as indentation unit. This is usually "    " (4 spaces) or "\t" (1
+# tab).
+indent-string='    '
+
+# Number of spaces of indent required inside a hanging or continued line.
+indent-after-paren=4
+
+# Expected format of line ending, e.g. empty (any line ending), LF or CRLF.
+expected-line-ending-format=LF

```

Comment:
Do we want to force LF line endings?

Reply:
If I remember correctly I think that by default git automatically converts line endings when it downloads the files(LF on Unix & CRLF on Windows) so it shouldn't matter.

Only one I know it matters is the entry point script for Docker has to be LF line endings, for when we use GitHub Actions